src/python/pants/explorer/server/BUILD

```diff
@@ -1,36 +0,0 @@
-# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
-# Licensed under the Apache License, Version 2.0 (see LICENSE).
-
-python_sources()
-
-python_distribution(
-    name="dist",
-    provides=python_artifact(
-        name="pantsbuild.pants.explorer-server",
-        description="Backend API server implementation for the Pants Explorer UI.",
-        classifiers=["Topic :: Software Development"],
-    ),
-    entry_points={
-        "pantsbuild.plugin": {
-            "rules": "pants.explorer.server.register:rules",
-        }
-    },
-)
```

Comment:
Note I didn't carry this over, since we aren't publishing the distribution and I didn't want any confusion